Streetcode/Streetcode.DAL/Entities/AdditionalContent/Subtitle.cs

```diff
@@ -21,10 +21,10 @@ public class Subtitle
     [MaxLength(50)]
     public string LastName { get; set; }
 
-    [Column(TypeName = "text")]
+    [Column(TypeName = "nvarchar(max)")]
```

Comment:
@LanchevychMaxym pick me up if we need to use `ntext` column type

Reply:
We have already change it